SPECS/conmon/conmon.spec

```diff
@@ -0,0 +1,399 @@
+%global with_debug 1
+%global with_check 0
+%if 0%{?with_debug}
+%global _find_debuginfo_dwz_opts %{nil}
+%global _dwz_low_mem_die_limit 0
+%else
+%global debug_package %{nil}
+%endif
+Summary:        OCI container runtime monitor
+Name:           conmon
+Version:        2.0.29
+Release:        3%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/containers/conmon
+#Source0: https://github.com/containers/conmon/archive/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+BuildRequires:  gcc
+BuildRequires:  git
+BuildRequires:  glib2-devel
+BuildRequires:  go-md2man
+BuildRequires:  make
+BuildRequires:  systemd-devel
+BuildRequires:  systemd-libs
+Requires:       glib2
+Requires:       systemd-libs
+
+%description
+%{summary}.
+
+%prep
+%autosetup -Sgit -n %{name}-%{version}
+
+%build
+make GOMD2MAN=go-md2man DEBUGFLAG="-g" bin/conmon
+make GOMD2MAN=go-md2man -C docs
+
+%install
+make PREFIX=%{buildroot}%{_prefix} install install.crio
+
+#define license tag if not already defined
+%{!?_licensedir:%global license %doc}
```

Comment:
Can be removed - we always have `%license` defined and we shouldn't overwrite it.